### DISCUSSION TITLE
Webpage at .vercel.app domain still seen after deleting

### DISCUSSION THREAD

**Discussion by omersuve** [vercel/community#7057] [Help]
I deleted my project, and after that I deleted my account as well. But the domain at <project_name>.vercel.app still shows the webpage.

What should I do to remove it?

Is it going to be removed after waiting some time or is it about something else?

**Comment by versecafe** [comment]
@omersuve if you just deleted it you may be hitting a mix of browser & Vercel caches that should stop overtime, how long has it been since you deleted everything?

**Reply by omersuve** [reply]
Hello @versecafe,
Its been more than 4 days and even though I deleted whole account with deleting github deployments, .verce.app website is still online, what should I do? I want to remove it and make it offline.
Please help.

**Comment by omersuve** [comment]
I has been 6 hours since I deleted.
